src/main/java/ua/com/juja/microservices/teams/slackbot/controller/TeamSlackbotController.java

```diff
@@ -46,48 +45,34 @@ public void onReceiveSlashCommandActivateTeam(@RequestParam("token") String toke
                                                   @RequestParam("text") String text,
                                                   @RequestParam("response_url") String responseUrl,
                                                   HttpServletResponse response) throws IOException {
-        log.debug("Received slash command 'Activate team' from user: '{}' text: '{}' token: '{}' response_url: '{}'",
-                fromUser, text, token, responseUrl);
         exceptionsHandler.setResponseUrl(responseUrl);
         if (isTokenCorrect(token, fromUser, response, "Activate team")) {
-
             sendInstantResponseMessage(response, ACTIVATE_TEAM_MESSAGE);
-
-            log.debug("Started activate team request to Team service. Text: '{}'", text);
-            Team activatedTeam = teamService.activateTeam(text);
-            log.debug("Finished activate team in Team service. New Team: '{}'", activatedTeam.toString());
-
+            teamService.activateTeam(text);
```

Comment:
If Exception thrown then the message in the slack on the impossibility to create a team, or in a slack message is sent that the team created. Return Team does not need at that time.